language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/command/visitors/FunctionCallExpressionTypeFinder.java

```diff
@@ -424,6 +428,13 @@ public void visit(WhileStatementNode whileStatementNode) {
 
     @Override
     public void visit(ConditionalExpressionNode conditionalExpressionNode) {
+
+        if (this.functionCallExpressionNode != null &&
```

Comment:
We don't need the null check here. Because due to the approach we are using this visitor, `functionCallExpressionNode` cannot be `null`.

Reply:
Removing `null` condition results spot bugs

Reply:
No problem then